TKeazirian.HTTPServer/Response/StatusMessage.cs

```diff
@@ -0,0 +1,23 @@
+namespace TKeazirian.HTTPServer.Response;
+
+public static class StatusMessage
+{
+    public static string GetStatusMessage(int statusCode)
+    {
+        string message;
+        switch (statusCode)
+        {
+            case (int)HttpStatusCode.Ok:
```

Comment:
why do you have to cast to `int` here?

Reply:
`HttpStatusCode.Ok` will give me the string value.  Casting it to an `int` gives me the integer value.